### PR TITLE
Moves changelog API call from build cycle to clientside..

### PR DESCRIPTION
**Pull request by kimblim**
.. with a link to Github for fallback

### CODE REVIEW

File: fractal-theme/assets/js/src/changelog.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// Create the XHR object.
+function createCORSRequest(method, url) {
+  const xhr = new XMLHttpRequest();
+  if ("withCredentials" in xhr) {
```
Comment (grandorf) [inline fractal-theme/assets/js/src/changelog.js]: single quotes please

File: fractal-theme/assets/js/src/changelog.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+// Create the XHR object.
+function createCORSRequest(method, url) {
+  const xhr = new XMLHttpRequest();
+  if ("withCredentials" in xhr) {
+    // XHR for Chrome/Firefox/Opera/Safari.
+    xhr.open(method, url, true);
+  } else if (typeof XDomainRequest != "undefined") {
```
Comment (grandorf) [inline fractal-theme/assets/js/src/changelog.js]: single quotes please

File: fractal-theme/assets/js/src/changelog.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  } else {
+    // CORS not supported.
+    xhr = null;
+  }
+  return xhr;
+}
+
+// Make the actual CORS request.
+function makeCorsRequest(url) {
+  let apiUrl = url;
```
Comment (grandorf) [inline fractal-theme/assets/js/src/changelog.js]: 'apiUrl' is never reassigned. Use 'const' instead

File: fractal-theme/assets/js/src/changelog.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+// Create the XHR object.
+function createCORSRequest(method, url) {
+  const xhr = new XMLHttpRequest();
```
Comment (grandorf) [inline fractal-theme/assets/js/src/changelog.js]: xhr looks like it's reassigned. Use 'let' instead.

File: fractal-theme/assets/js/src/changelog.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+  xhr.onerror = function() {
+    console.log('There was an error making the request.');
+  };
+
+  xhr.send();
+}
+
+function createInnerHTML(obj) {
+  let htmlString = ``;
```
Comment (grandorf) [inline fractal-theme/assets/js/src/changelog.js]: use single quotes. Only use back-ticks when needed for template strings

File: fractal-theme/assets/js/src/changelog.js [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+  const innerHTMLTarget = document.querySelector('.changelog');
+
+  obj.forEach((release) => {
+    const timestamp = new Date(Date.parse(release.published_at.toLocaleString('de-DE'))).toUTCString();
```
Comment (grandorf) [inline fractal-theme/assets/js/src/changelog.js]: don't you mean da-dk?

File: fractal-theme/assets/js/src/changelog.js [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+    htmlString +=
+      `<article class="changelog__item">
+        <figure class="changelog__avatar">
+          <img width="60" height="60" src="${release.author.avatar_url}" alt="Authored by ${release.author.login}" />
+        </figure>
+        <div class="changelog__details">
+            <h2 class="changelog__title"><a href="${release.html_url}">${release.name}</a> (${release.tag_name})</h2>
```
Comment (grandorf) [inline fractal-theme/assets/js/src/changelog.js]: stick with two spaces even if it's a html template string

File: fractal-theme/assets/js/src/changelog.js [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+            <p class="changelog__desc">${release.body}</p>
+            <span class="changelog__released">${timestamp}</span>
```
Comment (grandorf) [inline fractal-theme/assets/js/src/changelog.js]: maybe use <time> element here?

File: src/docs/03-Code Guidelines/06-Changelog.md [hunk 9]
```diff
@@ -1,16 +1,5 @@
...
-                <img width="60" height="60" src="{{this.author.avatar_url}}" alt="Authored by {{this.author.login}}" />
-            </figure>
-            <div class="changelog__details">
-                <h2 class="changelog__title"><a href="{{this.html_url}}">{{this.name}}</a> ({{this.tag_name}})</h2>
-                <p class="changelog__desc">{{this.body}}</p>
-                <span class="changelog__released">{{this.published_at}}</span>
-            </div>
-        </article>
-        {{/each}}
+        <p>See a <a href="https://github.com/youseedk/dna/releases">complete list of releases</a> on GitHub.</p>
```
Comment (grandorf) [inline src/docs/03-Code Guidelines/06-Changelog.md]: Nice! Maybe add `target="_blank"` here

Comment (kimblim) [reply src/docs/03-Code Guidelines/06-Changelog.md]: I personally hate it when websites make that decision for me — I prefer not to have it.

Comment (grandorf) [reply src/docs/03-Code Guidelines/06-Changelog.md]: Ok fine with me

**Pull request merged**